homeassistant/components/xiaomi_miio/light.py

```diff
@@ -938,3 +964,109 @@ async def async_update(self):
     async def async_set_delayed_turn_off(self, time_period: timedelta):
         """Set delayed turn off. Unsupported."""
         return
+
+
+class XiaomiGatewayLight(LightEntity):
+    """Representation of a gateway device's light."""
+
+    def __init__(self, gateway_device, gateway_name, gateway_device_id):
+        """Initialize the XiaomiGatewayLight."""
+        self._gateway = gateway_device
+        self._name = f"{gateway_name} Light"
+        self._gateway_device_id = gateway_device_id
+        self._unique_id = f"{gateway_device_id}-light"
+        self._available = False
+        self._is_on = None
+        self._brightness_pct = 100
+        self._rgb = (255, 255, 255)
+        self._hs = (0, 0)
+
+    @property
+    def unique_id(self):
+        """Return an unique ID."""
+        return self._unique_id
+
+    @property
+    def device_id(self):
```

Comment:
This doesn't seem used. Please remove the property.

Reply:
done